docs/maui/views/MediaElement.md

```diff
@@ -151,6 +204,29 @@ By default, the media that is defined by the `Source` property doesn't immediate
 
 Platform provided media playback controls are enabled by default, and can be disabled by setting the `ShouldShowPlaybackControls` property to `false`.
 
+### Using MetaData
+
+A `MediaElement` can use metadata for `MediaElement.MetaDataTitle`, `MediaElement.MetaDataArtist` and `MediaElement.MetaDataArtworkUrl` You can set
+the title or artist to show what is currently playing on lockscreen controls for Windows, Mac Catalyst, IOS, and Android. You can set a local or 
+remote URL with artwork for the lockscreen. It should be at least 1080P for best quality to be displayed. It cannot be a local file, or resource. It
+must be a URL and be either `.jpg` or `.png` For title or artist the only requirement is that it be a string. This can be set in code behind or in xaml.
+
+```xaml
+<toolkit:MediaElement MetaDataTitle="Title"
+                MetaDataArtist="Artist"
+                MetaDataArtworkUrl="http://www.a-domain.com/image.jpg" />
```

Comment:
@bijington Shaun, for the documentation is it OK to use a-domain.com or do we have a placeholder domain we need to follow? 😃

Reply:
I updated it to a domain from google used for project documentation examples.